components/31-molecules/partner-block/partner-block.twig

```diff
@@ -0,0 +1,79 @@
+<div class="partner-block box-no-icon {{ modifier }}">
+  {% if title %}
+    {% include '@heading' with {
+      'type': 'h3',
+      'heading_text': title
+    } %}
+  {% endif %}
+
+  {% if description %}
+    {% include '@paragraph' with {
+      'text': description
+    } %}
+  {% endif %}
+
+
+  {% if single_partner %}
+    <div class="single-partner">
+      {% include '@image' with {
+        src: '/styleguide/img/stock/ocmw-logo.svg' | path,
```

Comment:
Would be nice if you made these configurable and looped over them in the template. That'd make the template way more readable in my opinion.

(but that's a nice-to-have)

Reply:
Done.